### ISSUE TITLE
Finalização do TCC

### ISSUE THREAD

**Issue by HaraHeique** [HaraHeique/TCC-rede-neural-siamesa#44]

**Comment by HaraHeique** [comment]
### Organização dos datasets para as 20 rodadas

#### Treinamento

accuracy,val_accuracy,loss,val_loss
1 - arr_accur1,arr_val_accu1,loss_val_val1,val_loss1
2 - arr_accur2,arr_val_accu2,loss_val_val2,val_loss2
...

#### Predição (teste)

f-f,f-h,f-r,h-f,h-h,h-r,r-f,r-h,r-r
1 - 12,32,43,54,12,43,54,12,43
2 - 12,32,43,54,12,43,54,12,43
...

**Comment by HaraHeique** [comment]
## Principais hyperparâmetros definidos pelo _hyperas_

Best performing model chosen hyperparameters:
{'Activation': 'selu', 'Dropout': 0.24697647633830466, 'Dropout_1': 0.8228431108922754, 'Dropout_2': 0.06536980304050743, 'LSTM': 128, 'activation': 'softsign', 'batch_size': 64, 'clipnorm_choice': 1.5, 'epochs': 20, 'kernel_initializer_choice': <tensorflow.python.keras.initializers.initializers_v2.GlorotNormal object at 0x0000019208BD6048>, 'loss_choice': <tensorflow.python.keras.losses.MeanSquaredError object at 0x0000019207F0CB70>, 'lr_choice': 0.1, 'optimizer_choice_option': 'Adadelta', 'recurrent_activation': 'hard_sigmoid', 'recurrent_activation_1': 'sigmoid'}

**Comment by HaraHeique** [comment]
#### Comando para rodar o main.py como módulo de bootstrap do código adaptado na branch **test/experimentos-server-ifes** para ser executado via ssh no server do ifes (pode ser rodado no visual code também):

> python3 -m src/main
